ahmed-yara-sara/src/components/Search.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component, useEffect ,useState } from "react";
+import "react-bootstrap";
+import { Spinner, Form, FormControl, Button } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
+import DropdownMenu from "./Dropdown"
+export default function Search(props) {
+  const [isSubmitted, submitForm] = React.useState(false);
+  const [searchValue, setSearchValue] = React.useState("");
+  const [category, setCategory] = useState({});
+  const handleChange = (event) => {
+    setSearchValue(event.target.value);
+  };
+  const changeCategory = (category) => {
+    console.log(category);
+    //props.setIsLoading(true);
```

Comment:
again try not to leave extra comments and console.logs in your final version of code